packages/signers/src/transaction-signer.ts

```diff
@@ -30,7 +30,7 @@ export type TransactionSigner<TAddress extends string = string> =
  * import { isTransactionSigner } from '@solana/signers';
  *
  * const address = '1234..5678' as Address<'1234..5678'>;
- * isTransactionSigner({ address, signTransaction: async () => {} }); // true
+ * isTransactionSigner({ address, signTransactions: async () => {} }); // true
  * isTransactionSigner({ address, modifyAndSignTransaction: async () => {} }); // true
  * isTransactionSigner({ address, signAndSendTransaction: async () => {} }); // true
```

Comment:
Thanks, could you pluralize these ones as well whilst you're at it?

Reply:
sure, updated in 3332bc0ef207ca6577f6cf06c089d4c6c5a48427